ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -49,6 +50,7 @@ function RJSFForm(props) {
     ArrayFieldTemplate = MesheryArrayFieldTemplate,
     ObjectFieldTemplate = MesheryCustomObjFieldTemplate,
     // WrapIfAdditionalTemplate = MesheryWrapIfAdditionalTemplate,
+    FieldErrorTemplate= MesheryFieldErrorTemplate,
```

Comment:
What does this do, what changes it introduces over existing?

Reply:
Earlier we were using FieldTemplate in v3, but due to newer v5 we can separate out the control of the error msg with Field Error Template.